app/controllers/issues_controller.rb

```diff
@@ -0,0 +1,26 @@
+class IssuesController < ApplicationController
+
```

Comment:
@UsmanAhmad169 's issue-PR once gets merged into develop, reverse merge this branch in peer-programming with him so it is done quickly. @hammad-Ikhlaq-7vals

Reply:
ok